src/frontend/sharedComponents/ProjectSettingsCard.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {StyleSheet, View, ViewStyle} from 'react-native';
+import {WHITE, VERY_LIGHT_GREY, BLACK} from '../lib/styles';
+
+interface CardProps {
+  children: React.ReactNode;
+  style?: ViewStyle;
+  backgroundColor?: string;
+  borderColor?: string;
+}
+
+export const ProjectSettingsCard = ({
+  children,
+  style,
+  backgroundColor = WHITE,
+  borderColor = VERY_LIGHT_GREY,
```

Comment:
these are not being used, so i don't think it is necessary. If that the case, this is just a View with a child and some styling.

I think so we are not unnecessarily just forwarding a compnent, lets just get rid of this component, and move the `View` and `Styles` up to the SettingsCardRow